smithy-model/src/test/java/software/amazon/smithy/model/ErrorsFileTest.java

```diff
@@ -17,6 +17,7 @@
 
 import java.util.concurrent.Callable;
 import java.util.stream.Stream;
+
```

Comment:
On `singleFileTestSource()`, I agree that it is not used. It is an extremely useful method when debugging a specific model to separate it from every other model which I did for these changes and others, and can see myself attempting to do pretty much for any model change. I think it should stay there as a way to prevent future devs from having to figure out how to isolate their debugging scenarios for a particular file.

On the new empty line: I'll remove. It looks like I organized imports it at some point when edits were made to the file.